app/views/users/show.html.erb

```diff
@@ -1,2 +1,43 @@
-<h1>Specific User details</h1>
-<p>Specific user's details will appear here</p>
+<h1>User details</h1>
+<h3 class="user-id">User ID: <%= @user.id %></h3>
+<main id="user-index" class="page">
+<div id="user-page">
+<div class="back-link-container">
+        <%= link_to "Home", root_path, class: "back-link" %>
+        <%= link_to "Back", :back, class: "back-link" %>
+      </div>
+      <br>
+      <div class="user-container">
+      <div class="image-div">
+<%= image_tag('user.png',  :width => "100px", :heigth=>"100px") %>
+</div>
+<div class="userdetails-div">
+<h3><%= @user.name %></h3>
+<p class="posts-counter">Number of posts: <%= @user.posts_counter%></p>
+</div>
+</div>
+<article class="user-bio">
+      <h3>Bio</h3>
+      <div class="bio"><%= @user.bio %></div>
+    </article>
+
+<article class="post-article">
+<% @posts.each do |post| %>
+<div class="post-container">
```

Comment:
- [ ] It would be nice if you use `recent_posts` method here which you have created in the models to follow the project requirements 😊

Reply:
The above project requirement has been catered in users_controller.rb file which is calling the recent_posts method there def show
    @user = User.find_by_id(params[:id])
    @posts = @user.recent_posts
  end.
Here we are just looping through the 3 recent posts fetched. This project requirement has been catered in users controller. 
Screen shot attached below for your kind reference.
![image](https://github.com/baqar-abbas/blog-app-rails/assets/117091923/5aae0b41-0abd-4046-b96b-deacd41188e9)